src/xdocs/config_coding.xml

```diff
@@ -753,6 +753,28 @@ String nullString = null;
           shows that the developer isn't really confident that Java
           initializes instance variables to default values.
         </p>
+
+        <p>
+          <b>onlyObjectReferences</b> - This option allows class or object members to
```

Comment:
This is still the same issue as before because you have `<b>onlyObjectReferences</b>` and it is still a re-hash of the properties table with nothing new added.
I would just remove this.